<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BPRRLimitRemover</class>
 <widget class="QDialog" name="BPRRLimitRemover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Road Rule Limit Remover</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RRLimitRemover.qrc">
    <normaloff>:/BPRoadRuleLimitRemover/rrlr_icon.ico</normaloff>:/BPRoadRuleLimitRemover/rrlr_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lnExecutable">
          <property name="placeholderText">
           <string>Select executable...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnBrowse">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="chkTime">
        <property name="text">
         <string>Remove minimum time limit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkShowtime">
        <property name="text">
         <string>Remove maximum showtime limit</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>125</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnApply">
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnQuit">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="RRLimitRemover.qrc"/>
 </resources>
 <connections/>
</ui>
